### ISSUE TITLE
How is the wwwroot folder mapped as a part of this process.

### ISSUE THREAD

**Issue by Itsey** [aspnet/AspNetCore.Docs#15155]
Hi I have done this a couple of times now but it appears that the folder is mapped to the root folder of the solution but when debugging in the iisexpress profile the root folder appears to be mapped to the wwwroot folder of the project directory. The basic debugging experience works but the styles and js files are mismatched.  Is there something different we need to do to map the folder to the wwwroot folder?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f5477f20-bdbf-ae0b-63b4-8bf6789f7a48
* Version Independent ID: fa9dc362-76a4-5d11-f457-b3d8550f9997
* Content: [Development-time IIS support in Visual Studio for ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-3.0#feedback)
* Content Source: [aspnetcore/host-and-deploy/iis/development-time-iis-support.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/iis/development-time-iis-support.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by Itsey** [comment]
Ignore this - it appears that the issue I am seeing is not as simple as the question I have raised here - I neeed to look more closely and only raise it if I still dont understand it.

**Comment by guardrex** [comment]
That's the correct behavior for debugging; and although the *web root* can be changed ...

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-3.0#webroot

... it's based on the *content root* path, and the content root still needs to point to the project's root folder for debugging.

Why do you need to use the published *wwwroot* assets in testing when the published assets are just what's the in project's *wwwroot* folder?

**Comment by Itsey** [comment]
Hi sorry I don't understand this and I realised that the error I'm seeing (missing images) is not actually what I thought it was.  I looked at the IIS path and saw it wasn't mapped to wwwroot and assumed (Wrongly) that this was the cause.  However the actual issue I'm seeing is more complex and I think it is based on my lack of understanding not a fault with the docs.  I tried to close the issue but was too slow - thank you for your help :)
